Sources/Sentry/Public/SentryOptions.h

```diff
@@ -164,7 +164,7 @@ NS_SWIFT_NAME(Options)
 /**
  * Array of integrations to install.
  */
-@property (nullable, nonatomic, copy) NSArray<NSString *> *integrations;
+@property (nullable, nonatomic, readonly) NSArray<NSString *> *integrations;
```

Comment:
h: Why did we changed this in here?
Integrations can be manipulated by users to remove something they don’t like.

Reply:
I wanted to be able to mutate the list in the implementation.

I misunderstood `copy`, thinking it _hands out copies on access_, but it actually _makes copies on assignment_... that's ok, because I should be able to make these readwrite/copy and it should still work for my purposes.

Reply:
Hm, sadly I cannot make it a private readwrite NSMutableArray and public copy nonmutable NSArray

Reply:
I found another way to do it